### DISCUSSION TITLE
Remote MCDU never get work

### DISCUSSION THREAD

**Discussion by bigjohnx** [flybywiresim/aircraft#7547] [Help]
I am using FBW installer to install A32NX plus simbridge.
As the simbridge have a built in web server to provide remote MCDU access.
I had tried to open but never get working.
Just show a MCDU UI without screen display content, and click on button no any response.
How to troubleshoot?
A32NX version: V0.8.2
Simbridge version: V0.3.16
![rmcdu](https://user-images.githubusercontent.com/8724900/194913573-159f7f6b-f3ed-485a-b764-859be68980b9.png)

**Comment by ExampleWasTaken** [comment]
Check that the port set in the flypad settings match the port set in simbridge.

**Reply by ExampleWasTaken** [reply]
More info here: https://docs.flybywiresim.com/simbridge/troubleshooting/
